app/controllers/frontend/content_controller.rb

```diff
@@ -1,4 +1,6 @@
 class Frontend::ContentController < Frontend::ApplicationController
+  before_action :set_field_config
+
   def index
     @screen = Screen.find(params[:screen_id])
```

Comment:
Now that we have set_field_config, should we remove these @screen and @field lines as duplicative?